src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -4049,6 +4055,7 @@ private bool HasCustomInitializer(XamlObjectDefinition definition)
                     return true;
                 }
 
+                propertyType = GetUnderlyingType(propertyType);
```

Comment:
Should it be placed immediately above the test? That way, we'd support nullable enums as well. If so, could you also add a test for nullable enums ?

Reply:
You are right, I made these changes.